sdk/python/jobs/pipelines/1d_pipeline_with_non_python_components/pipeline_with_non_python_components.ipynb

```diff
@@ -123,9 +123,11 @@
     "    command=\"curl ${{inputs.url}} > ${{outputs.output_folder}}/${{inputs.file_name}}\",\n",
     "    environment=environment,\n",
     "    inputs=dict(url=download_url, file_name=file_name),\n",
-    "    # example how to change path of output on step level, \n",
```

Comment:
why this file changed?

Reply:
smoke/build will conduct black check, and I formatted files detected by black to make smoke/build CI passed. So, this file changed